ts/src/header-validator/validate-json.ts

```diff
@@ -1527,6 +1638,14 @@ function trigger(ctx: RegistrationContext, j: Json): Maybe<Trigger> {
           (ctx, j) => triggerContextID(ctx, j, aggregatableSourceRegTimeVal),
           null
         ),
+        attributionScopes: field(
+          'attribution_scopes',
+          (ctx, j) => {
+            const scopes = attributionScopes(ctx, j, Infinity)
+            return scopes?.value?.size === 0 ? None : scopes ?? None
```

Comment:
This doesn't emit an error when the list is empty. Let's pass both min length and max length as arguments to `attributionScopes` instead.

Reply:
See other comments for this, we are now accepting empty list but treating it the same as null.

Reply:
`None` is an error. Please take a closer look.

Reply:
Thanks for the input. Done.